abi.html

```diff
@@ -5675,6 +5675,7 @@ <h4><a href="#expressions">5.1.6 Expressions</a></h4>
                ::= pp_ &lt;<a href="#mangle.expression">expression</a>&gt;                                     # prefix ++
                ::= mm_ &lt;<a href="#mangle.expression">expression</a>&gt;                                     # prefix --
                ::= cl &lt;<a href="#mangle.expression">expression</a>&gt;+ E                                   # expression (expr-list), call
+               ::= cp &lt;<a href="#mangle.base-unresolved-name">base-unresolved-name</a>&gt; &lt;<a href="#mangle.expression">expression</a>&gt;* E            # (name) (expr-list), call that would use argument-dependent lookup but for the parentheses
```

Comment:
I've added the paragraph. I indicated that the reference is to C++11 because the paragraph number for the rule was different before and has since changed. The wording around the block scope case was not present in C++03.